### PR TITLE
Update copy on /solutions/ai

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Update copy on /solutions/ai based on [copydoc](https://docs.google.com/document/d/1AtAApqxX9cbWf4a3cakEYbk7OoTfdBBi7FlA3WZz5ZM/edit)
- Formats file, updated sections have comments

## QA

- Go to https://canonical-com-1246.demos.haus/solutions/ai and check the changes match the [copydoc](https://docs.google.com/document/d/1AtAApqxX9cbWf4a3cakEYbk7OoTfdBBi7FlA3WZz5ZM/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10536

### CODE REVIEW

File: templates/solutions/ai/index.html [hunk 1]
```diff
@@ -1,114 +1,145 @@
...
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>One platform from workstations to edge deployments</h2>
+      </div>
+      <div class="col">
+        <p>
+          Ubuntu is the OS of choice for data scientists. Develop machine learning models on Ubuntu workstations and deploy to Ubuntu servers. Continue with the same familiar Ubuntu experience throughout the AI lifecycle, across clouds and all the way to the edge.
```
Comment (petesfrench) [inline templates/solutions/ai/index.html:136]: Change here

File: templates/solutions/ai/index.html [hunk 2]
```diff
@@ -297,50 +342,61 @@ <h2>Optimised for silicon</h2>
...
+      <div class="col">
+        <p>
+          Drive real-time insight in the field and on the factory floor by running your AI models directly on edge devices.
+        </p>
+        <p>
+          Deploy compute where you need it to power AI use cases at the edge. Solve security, management and update challenges for your fleet with an OS purpose-built for IoT devices and optimised for AI.
+        </p>
+        <hr />
+        <p>
+          <a href="/engage/guide-to-deploying-ai-at-the-edge">Edge AI explained&nbsp;&rsaquo;</a>
```
Comment (petesfrench) [inline templates/solutions/ai/index.html:365]: Change here

Comment (bartaz) [reply templates/solutions/ai/index.html:365]: This link seems to be 404: https://canonical-com-1246.demos.haus/engage/guide-to-deploying-ai-at-the-edge

Comment (bartaz) [reply templates/solutions/ai/index.html:365]: I think the link is on ubuntu.com, not canonical.

**Review by bartaz — APPROVED** [review]
LGTM

**Pull request merged**